python/sglang/srt/disaggregation/decode.py

```diff
@@ -133,8 +135,18 @@ def _init_kv_manager(self) -> BaseKVManager:
 
     def add(self, req: Req) -> None:
         """Add a request to the pending queue."""
-
-        kv_receiver_class = get_kv_class(self.transfer_backend, KVClassType.RECEIVER)
+        if (
+            req.bootstrap_room == FakeBootstrapRoom
+            and req.bootstrap_host == FakeBootstrapHost
+        ):
+            # Fake transfer for warmup reqs
+            kv_receiver_class = get_kv_class(
+                self.transfer_backend, KVClassType.RECEIVER, fake_transfer=True
```

Comment:
can we just have a "fake" transfer_backend and get rid of `fake_transfer`?

Reply:
good point .